doc/api.yml

```diff
@@ -2429,6 +2429,19 @@ components:
           type: integer
         number_of_months:
           type: integer
+        fees:
+          type: array
+          items:
+            $ref: '#/components/schemas/FeeTypeComponent'
+    FeeTypeComponent:
+      type: object
+      properties:
+        id:
+          type: string
+        month_number:
+          type: integer
```

Comment:
add description.

i think it is even better if you add an enum for months.
From January to December